posts/zzz_DO_NOT_EDIT_introducing.../introducing_sdtm.oak.qmd

```diff
@@ -0,0 +1,252 @@
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+{sdtm.oak} v0.1 is now available on [CRAN](https://cran.r-project.org/web/packages/sdtm.oak/index.html).
+In this blog post, we will introduce the package, key concepts, and examples. {sdtm.oak} is developed in collaboration with volunteers from several companies, including Roche, Pfizer, GSK, Transition Technologies Science, and Atorus Research. {sdtm.oak} is also sponsored by CDISC COSA with a vision of being part of CDISC 360 to address end-to-end standards development and implementation.
+
+# Filling the Gap
+
+{sdtm.oak} package addresses a critical gap in the Pharmaverse suite by enabling study programmers to create SDTM datasets in R, complementing the existing capabilities for ADaM and TLGs.
+
+Let's explore the challenges with SDTM programming.
+Although SDTM is simpler with less complex derivations compared to ADaM, it presents unique challenges.
+Unlike ADaM, which uses SDTM datasets as its source with a well-defined structure, SDTM relies on raw datasets as input.
+These raw datasets can vary widely in structure, depending on the data collection and EDC system used.
+Even the same eCRF, when designed in different EDC systems, can produce raw datasets with different structures.
+
+Another challenge is the variability in data collection standards.
+Although CDISC has established CDASH data collection standards, many pharmaceutical companies have their own standards, which can differ significantly from CDASH.
+Additionally, since CDASH is not mandated by the FDA, sponsors can choose the data collection standards that best fit their needs.
+
+There are hundreds of EDC systems available in the marketplace, and the data collection standards vary significantly.
+Creating a single open-source package to work with all sorts of raw data formats and data collection standards seemed impossible.
+But here's the good news: not anymore!
+The {sdtm.oak} team has a solution to address this challenge.
+
+{sdtm.oak} is designed to be highly versatile, accommodating varying raw data structures from different EDC systems and external vendors.
+Moreover, {sdtm.oak} is data standards agnostic, meaning it supports both CDISC-defined data collection standards (CDASH) and various proprietary data collection standards defined by pharmaceutical companies.
+The reusable algorithms concept in {sdtm.oak} provides a framework for modular programming, making it a valuable addition to the Pharmaverse ecosystem.
+
+# EDC & Data standards agnostic
+
+We adopted the following innovative approach to make {sdtm.oak} adaptable to various EDC systems and data collection standards:
+
+-   SDTM mappings are categorized as algorithms and developed as R functions.
+-   Used datasets and variables as parameters to function calls.
+
+# Algorithms
+
+The SDTM mappings that transform the collected source data (eCRF, eDT) into the target SDTM data model are grouped into algorithms.
```

Comment:
Should we explain the acronyms eCRF and eDT?

Reply:
explained acronyms